src/flash/nor/stm32h7x.c

```diff
@@ -824,6 +824,14 @@ static int stm32x_probe(struct flash_bank *bank)
 			flash_size_in_kb /= 2;
 		break;
 	case 0x483:
+		/* For STM32H72x/73x
+		 *  - STM32H7xxxG devices contains single bank, 1024 Kbyte
+		 */
+		if (flash_size_in_kb == 128)
+			has_dual_bank = false;
+		else
+			/* flash size is 2M or 1M */
+			flash_size_in_kb /= 1;
```

Comment:
what's the point of the else?  divide `/1` seems unnecessary.

Reply:
You're right. My copy-paste mistake. Changed now.